src/liboslexec/backendllvm.cpp

```diff
@@ -267,13 +268,25 @@ BackendLLVM::llvm_alloca(const TypeSpec& type, bool derivs,
 }
 
 
+bool
+BackendLLVM::is_stack_parameter(const Symbol& sym)
```

Comment:
I am wondering if it ending up on the stack is the results, and the query here is more like "param_could_be_local" or something indicating that the parameter to could be pro(de)moted from a parameter to a local variable because it has no upwards connections.

Reply:
Sure, makes sense to me.

Reply:
```
bool
BackendLLVM::can_treat_param_as_local(const Symbol& sym)
```

Reply:
This is pushed.